modules/subscriptions.php

```diff
@@ -630,11 +630,14 @@ function comment_subscribe_init() {
 
 		// Check if Mark Jaquith's Subscribe to Comments plugin is active - if so, suppress Jetpack checkbox
 
-		$str = '';
+		$unique_id = esc_attr( uniqid( 'jetpack-subscription-form-' ) );
```

Comment:
There's no need to call `esc_attr`, you're doing it properly already by escaping late - when using the value for output.